src/site/pages/digital/src/containers/ExprToCircuitPopup/index.tsx

```diff
@@ -0,0 +1,201 @@
+import {useState} from "react";
+
+import {CreateAddGroupAction} from "core/actions/addition/AddGroupActionFactory";
+
+import {Popup}   from "shared/components/Popup";
+
+import {CloseHeaderPopups} from "shared/state/Header";
+import {useSharedDispatch, useSharedSelector} from "shared/utils/hooks/useShared";
+
+import {OrganizeMinDepth} from "core/utils/ComponentOrganizers";
+
+import {GroupAction}             from "core/actions/GroupAction";
+import {CreateDeselectAllAction,
+        CreateGroupSelectAction,
+        SelectAction,}            from "core/actions/selection/SelectAction";
+import {PlaceAction}             from "core/actions/addition/PlaceAction";
+import {CreateICDataAction}      from "digital/actions/CreateICDataAction";
+
+import {DigitalCircuitInfo}     from "digital/utils/DigitalCircuitInfo";
+import {DigitalComponent}       from "digital/models/DigitalComponent";
+import {ICData}                 from "digital/models/ioobjects/other/ICData";
+import {IC}                     from "digital/models/ioobjects/other/IC";
+import {LED}                    from "digital/models/ioobjects/outputs/LED";
+import {ConstantLow}            from "digital/models/ioobjects/inputs/ConstantLow";
+import {ConstantHigh}           from "digital/models/ioobjects/inputs/ConstantHigh";
+import {Button}                 from "digital/models/ioobjects/inputs/Button";
+import {Switch}                 from "digital/models/ioobjects/inputs/Switch";
+import {Clock}                  from "digital/models/ioobjects/inputs/Clock";
```

Comment:
nit: Put a space b/t this import and from `utils`, the models can be vertically aligned just with themself